webpack/scenes/Content/Table/ContentTable.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useCallback, useEffect, useMemo } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { STATUS } from 'foremanReact/constants';
+import { TableVariant, Thead, Tbody, Tr, Th, Td } from '@patternfly/react-table';
+import TableWrapper from '../../../components/Table/TableWrapper';
+import { getContent, getContentTypes } from '../ContentActions';
+import { selectContent, selectContentStatus, selectContentError, selectContentTypes, selectContentTypesStatus } from '../ContentSelectors';
+import SelectableDropdown from '../../../components/SelectableDropdown';
+import contentConfig from '../ContentConfig';
+import Loading from '../../../components/Loading';
+
+const ContentTable = () => {
+  const dispatch = useDispatch();
+  const status = useSelector(selectContentStatus);
+  const error = useSelector(selectContentError);
+  const response = useSelector(selectContent);
+  const enabledTypesResponse = useSelector(selectContentTypes);
+  const enabledTypesStatus = useSelector(selectContentTypesStatus);
+  const [contentTypeSelected, setContentTypeSelected] = useState('Python Packages');
+  const [searchQuery, updateSearchQuery] = useState('');
+  const [columnHeaders, setColumnHeaders] = useState([]);
+  const [contentTypes, setContentTypes] = useState(null);
+
+  const { results, ...metadata } = response;
+
+  function buildRows(details) {
+    const rows = [];
+    if (details) {
+      columnHeaders.forEach(header =>
+        rows.push(<Td>{header.getProperty(details)}</Td>));
+    }
+    return rows;
+  }
+
+  useEffect(() => {
+    dispatch(getContentTypes());
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  useEffect(() => {
+    if (contentTypes) {
+      setColumnHeaders(contentConfig().find(type =>
+        type.names.singular === contentTypes[contentTypeSelected][0]).columnHeaders);
+    }
+  }, [contentTypes, contentTypeSelected]);
+
+  useMemo(() => {
+    if (enabledTypesStatus === STATUS.RESOLVED) {
+      const types = {};
+      contentConfig().forEach((type) => {
+        if (enabledTypesResponse.includes(type.names.singular)) {
+          types[type.names.title] = [type.names.singular, type.names.plural];
+        }
+      });
+      setContentTypes(types);
+    }
+  }, [enabledTypesResponse, enabledTypesStatus]);
+
+  const fetchItems = useCallback(
+    () => getContent(contentTypes[contentTypeSelected][1]),
+    [contentTypes, contentTypeSelected],
+  );
```

Comment:
```suggestion
  const fetchItems = useCallback(
    (params) => getContent(contentTypes[contentTypeSelected][1], params),
    [contentTypes, contentTypeSelected],
  );
```
Passing params to fetchItem will make pagination and search work..